src/App.css

```diff
@@ -36,3 +36,15 @@
     transform: rotate(360deg);
   }
 }
+
+.screen {
+    height: 300px;
+    width: 500px;
+    background-color: grey;
+    position: fixed;
+    top: 50%;
+    left: 50%;
+    /* bring your own prefixes */
```

Comment:
What does that comment meant?

Reply:
dear, Mat it means that you can add your own values.

Reply:
Please remove the unnecesary comments.